intro-OpenStack/ch_HeatTemplates.tex

```diff
@@ -67,14 +67,15 @@ \section{\gls{Heat Orchestration Template}s}\label{sec:glsh-orch-templ}
 \item[\texttt{resources}] This section contains all the resources used
   by the Stack.  In this case, there is just a single VM instance.
 \end{description}
-Optional additional sections are \lstinline{parameter_groups},
-\lstinline{outputs}, and \lstinline{conditions}.
-
-The official OpenStack template documentation at
-\url{https://docs.openstack.org/heat/latest/template_guide} contains a
-specification of the \textsc{hot} format, as well as information on
-how to describe the various types of resources in a template.
-\textsc{vsc} also provides some example templates in the repository
+Optional additional sections are \strong{\lstinline{parameter_groups}},
+\strong{\lstinline{outputs}}, and \strong{\lstinline{conditions}}.
+
+The
+`\href{https://docs.openstack.org/heat/latest/template_guide}{Template
```

Comment:
I chose not to add long url's in footnotes after all, because nobody will try to type such long url's anyway.  Instead, I opted for clickable links, with a description that hopefully allows people who are not looking at the tutorial on a screen to still find the links.